### PR TITLE
changes in rex ui tests

### PR DESCRIPTION
**Pull request by pondrejk**
related to the new rex wizard UI, requires https://github.com/SatelliteQE/airgun/pull/819

### CODE REVIEW

File: tests/foreman/ui/test_remoteexecution.py [hunk 1]
```diff
@@ -167,43 +156,37 @@ def test_positive_run_job_template_multiple_hosts_by_ip(
...
-            )
-            assert job_status['overview']['job_status'] == 'Success'
-            assert {host_job['Host'] for host_job in job_status['overview']['hosts_table']} == set(
-                host_names
-            )
-            assert all(
-                host_job['Status'] == 'success'
-                for host_job in job_status['overview']['hosts_table']
-            )
+    clients = registered_hosts
```
Comment (JacobCallahan) [inline tests/foreman/ui/test_remoteexecution.py:159]: why the alias here?

**Review by omkarkhatavkar — APPROVED** [review]

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_remoteexecution.py -k test_positive_run_default_job_template_by_ip
airgun: 819

File: tests/foreman/ui/test_remoteexecution.py (hunk 1 above)
Comment (JacobCallahan) [reply tests/foreman/ui/test_remoteexecution.py:159]: ping @pondrejk :)

Comment (pondrejk) [reply tests/foreman/ui/test_remoteexecution.py]: updated, thanks

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**